cmd/config-host.go

```diff
@@ -27,6 +27,7 @@ var configHostCmd = cli.Command{
 	Name:   "host",
 	Usage:  "add, remove and list hosts in configuration file",
 	Action: mainConfigHost,
+	Before: initBeforeRunningCmd,
```

Comment:
I think no point in adding this, because initBeforeRunningCmd() will be called again in any of the sub-commands